File: src/uis/ReactWeb/src/entities/trip/models/TripParticipant.ts
import {User} from '@/entities/user/models/User.ts';
import {Trip} from '@/entities/trip/models/Trip.ts';
import {HuntingLicense} from '@/entities/huntinLicense/HuntingLicense.ts';

export interface TripParticipant {
    id?: string;
    participantId: string;
    participant?: User | null;
    huntingLicenseId: string;
    huntingLicense?: HuntingLicense | null;
    tripId: string;
    trip?: Trip | null;
}

File: src/services/Hunting/Modules/Document/Modules.Document.Core/Entities/TripParticipant.cs
using Modules.Document.Domain.Interfaces;

namespace Modules.Document.Domain.Entities
{
    public class TripParticipant : IBaseEntity
    {
        public Guid Id { get; set; }
        public Guid ParticipantId { get; set; }
        public User? Participant { get; set; }
        public Guid HuntingLicenseId { get; set; }
        public HuntingLicense? HuntingLicense { get; set; }
        public Guid TripId { get; set; }
        public Trip? Trip { get; set; }
        public bool Paid { get; set; }
    }
}
